### PR TITLE
feat - added avatars and badges

### PR DESCRIPTION
**Pull request by romabulani**
- Made directory structure for all the components
- will add the global variables in globalVar.css on the go
- Tried my best to reuse css and keep good class names, please let me know the improvments that can be done

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]
everything looks great especially those class names

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    font-size: 48px;
+    height: 6rem;
+    position: relative;
+    width: 6rem;
+}
+
+.avatar-md-size{
+    font-size: 32px;
+    height: 4rem;
+    position: relative;
```
Comment (manish591) [inline Components/avatar/avatar.css:42]: Why are you adding a position relative here and in all other images as well?

**Review by manish591 — COMMENTED** [review]
Your code is looking good. But if there is no use of position relative then you can remove this.

File: Components/avatar/avatar.css (hunk 1 above)
Comment (romabulani) [reply Components/avatar/avatar.css:42]: This same avatar.css I am importing in badges, so for that status badge, I have added postion-relative here and directly added position-absolute for badge

Comment (manish591) [reply Components/avatar/avatar.css:42]: Ok, got it

**Review by manish591 — APPROVED** [review]

**Review by Rohanmond — APPROVED** [review]
everything looks good, just one suggestion except of giving avatar-lg-size name you can put avatar-lg!! Otherwise ready for merging

**Comment by romabulani** [comment]
> everything looks good, just one suggestion except of giving avatar-lg-size name you can put avatar-lg!! Otherwise ready for merging

Thanks Rohan, changed the names

**Pull request merged**